diff --git a/setup.py b/setup.py
index 7e4b21f..ee7ce21 100644
--- a/setup.py
+++ b/setup.py
@@ -73,9 +73,9 @@ setup_kwds = dict(
 # change setup_kwds below before the next pkglts tag
 
 setup_kwds['setup_requires'] = ['openalea.deploy']
-build_prefix = "build-scons"
-setup_kwds['scons_scripts'] = ['SConstruct']
-setup_kwds['bin_dirs'] = {'bin': build_prefix + '/bin'}
+#build_prefix = "build-scons"
+#setup_kwds['scons_scripts'] = ['SConstruct']
+#setup_kwds['bin_dirs'] = {'bin': build_prefix + '/bin'}
 setup_kwds['entry_points']['wralea'] = ['alinea.caribu = alinea.caribu_wralea']
 setup_kwds['entry_points']["console_scripts"] = []
 setup_kwds['package_data'][''] = ['*.can', '*.R', '*.8', '*.opt', '*.light', '*.csv', '*.png','*.pyd', '*.so', '*.dylib']
