--- include/ANN/ANN.h	2010-01-28 04:40:01.000000000 +0000
+++ include/ANN/ANN.h	2017-06-28 17:17:33.000000000 +0100
@@ -76,7 +76,7 @@
   #ifdef DLL_EXPORTS
 	#define DLL_API __declspec(dllexport)
   #else
-	#define DLL_API __declspec(dllimport)
+	#define DLL_API __declspec(dllexport)
   #endif
   //----------------------------------------------------------------------
   // DLL_API is ignored for all other systems